src/test/java/com/github/tomakehurst/wiremock/matching/MatchesXPathPatternTest.java

```diff
@@ -320,7 +320,7 @@ void reportsErrorFromSubMatcher() {
             .match("<something>{ bad json</something>");
     checkJsonError(
         matchResult,
-        "Unexpected character ('b' (code 98)): was expecting double-quote to start field name\n at [Source: (String)\"{ bad json\"; line: 1, column: 4]");
+        "Unexpected character ('j' (code 106)): was expecting a colon to separate field name and value\n at [Source: (String)\"{ bad json\"; line: 1, column: 8]");
```

Comment:
I think we shouldn't change the way we parse user-supplied JSON in requests/responses, just stub mappings. I think maintaining a separate parser configuration and switching into it in the mappings loader (and possibly the API) might be the way forward here.